osf/migrations/0142_auto_20181030_1513.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-10-30 15:13
+from __future__ import unicode_literals
+from waffle.models import Flag
+from django.db import migrations
+from osf.features import EMBER_PROJECT_ANALYTICS
+
+def reverse_func(state, schema):
+    flag = Flag(name=EMBER_PROJECT_ANALYTICS, everyone=False)
+    flag.save()
+
+def remove_analytics_flag(state, schema):
+    Flag.objects.filter(name=EMBER_PROJECT_ANALYTICS).delete()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('osf', '0141_merge_20181023_1526'),
+    ]
+
+    operations = [
+        migrations.RunPython(remove_analytics_flag, reverse_func)
```

Comment:
We have a custom migration operation for this `osf.utils.migrations.DeleteWaffleFlags`. Can you use this instead?